arangod/Cluster/FollowerInfo.cpp

```diff
@@ -344,6 +344,10 @@ Result FollowerInfo::persistInAgency(bool isRemove) const {
   std::string planPath = PlanShardPath(*_docColl);
   AgencyComm ac;
   do {
+    if (_docColl->deleted() || _docColl->vocbase().isDropped()) {
```

Comment:
there is also the deleted state, is it sufficient to check this flag ?

Reply:
I think this check (as implemented) is sufficient.